devito/arch/compiler.py

```diff
@@ -1009,6 +1025,7 @@ def __contains__(self, k):
     'nvc++': NvidiaCompiler,
     'nvidia': NvidiaCompiler,
     'cuda': CudaCompiler,
+    'nvcc': CudaCompiler,
```

Comment:
'cuda': CudaCompiler,
    'nvcc': CudaCompiler,
```suggestion
    'nvcc': CudaCompiler,
    'cuda': CudaCompiler,
```